pages/common/license.md

```diff
@@ -0,0 +1,19 @@
+# license
+
+> Create license files for open-source projects.
+
+- Create a license:
```

Comment:
Isn't that the usual default behavior though? Especially with commands that have got a `-o` parameter.